mayastor/src/subsys/opts.rs

```diff
@@ -158,7 +159,7 @@ impl Default for TcpTransportOpts {
             ch2_success: true,
             max_qpairs_per_ctrl: 128,
             num_shared_buf: 511,
-            buf_cache_size: 32,
+            buf_cache_size: 64,
```

Comment:
Upstream default is still 32, is there a reason this was doubled?

Reply:
We have double the targets, or put differently we have the spdk one and our own. We can not get rid of the built-in one as then all the tests fail.

Reply:
OK, worth a code comment somewhere.

Reply:
Yeah, if it would stay this way then yes but we really need to clean that up IMHO.  I'll add comment there though.